src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTaskLogic.cs

```diff
@@ -447,6 +447,19 @@ private IEnumerable<OutputLibFile> InitLibFiles(IMSBuildItem[] libFiles)
                             string.Format(CultureInfo.CurrentCulture, Strings.InvalidPlatformVersion, targetFramework)
                         );
                     }
+                    var isNet5EraTfm = fw.Version.Major >= 5 &&
```

Comment:
Can we read from [this](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs#L94) property to check if it is a `net5.0 tfm`?

Reply:
We decided back when I added this that we _shouldn't_ expose this property outside of NuGet.Frameworks, so we have to do this whole dance manually every time :(